shoop/core/fields/__init__.py

```diff
@@ -52,12 +53,17 @@ def deconstruct(self):
         return (name, path, args, kwargs)
 
 
-class MoneyField(models.DecimalField):
+class CurrencyField(models.CharField):
```

Comment:
Even if the field itself might not have `choices`, it should generate a select-type `formfield` by default. That can be added later though.